### ISSUE TITLE
Accessibility: search results thumbnails have no alt-text

### ISSUE THREAD

**Issue by eporter23** [samvera/hyrax#6891]
### Descriptive summary

When using the WAVE plugin for Chrome browser, WAVE reports errors due to thumbnail images in search results having no alt text.  This was tested as part of the Sirenia docker image testing, but is probably not related to Sirenia specifically.

### Steps to reproduce the behavior in User Interface (UI)

1. Ensure that you have the WAVE plugin for Chrome
2. Load the search results page
3. Review the page in the WAVE report
4. Note that errors are shown for any thumbnails displayed next to results

### Actual behavior (include screenshots if available)

Hyrax 5.0.1 (main) - Sirenia docker image

"Linked image missing alternative text
An image without alternative text results in an empty link."

<img width="888" alt="Screenshot 2024-08-16 at 9 52 29 AM" src="https://github.com/user-attachments/assets/38b79f30-d740-463e-97a2-cba4d0bd285c">

### Acceptance Criteria/Expected Behavior

- [ ] Search results should not return errors related to alternative text during accessibility checks

### Rationale (for feature request only)

Provide the rationale or user story that describes "why" this issue should be addressed. Especially if this is a new feature or significant change to the existing implementation.

### Related work

Link to related issues or prior related work here.

**Comment by rjkati** [comment]
On dev.nurax (Hyrax 5.0.1), work thumbnails listed on the search results page now have alt-text per #6911, but collection thumbnails do not

https://dev.nurax.samvera.org/catalog?utf8=%E2%9C%93&locale=en&search_field=all_fields&q=
![collection-thumbnail](https://github.com/user-attachments/assets/cb7109fa-64ce-46eb-937f-c561243ecd79)
